src/pivot-table/components/DataGrid.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { memo, useCallback, useLayoutEffect } from "react";
+import { VariableSizeGrid, areEqual, GridOnItemsRenderedProps } from 'react-window';
+import { DataModel } from "../../types/types";
+import DataCell from './DataCell';
```

Comment:
mixing " and ' here

Reply:
Yeah I need to fix the linting. I've missed semi-colon on a few places as well

Reply:
OK I wont mind them then 👍